src/packages/@ncigdc/containers/RepositoryPage.js

```diff
@@ -265,6 +267,8 @@ export const RepositoryPageQuery = {
           cases {
             aggregations(filters: $filters aggregations_filter_themselves: false) {
               ${CaseAggregations.getFragment('aggregations')}
+            }
+            pieAggregations: aggregations(filters: $filters aggregations_filter_themselves: true) {
```

Comment:
this is how it is done on the exploration page? just want to make sure they are consistent